vcr/pe/presentation_definition_test.go

```diff
@@ -92,38 +44,109 @@ var testCredentialString = `
   }
 }`
 
+//go:embed test/*.json
+var testFiles embed.FS
+
+var pdJSONLD PresentationDefinition
+var pdJSONLDorJWT PresentationDefinition
+var pdJWT PresentationDefinition
+
+func init() {
+	if data, err := testFiles.ReadFile("test/pd_jsonld.json"); err != nil {
+		panic(err)
+	} else {
+		if err = json.Unmarshal(data, &pdJSONLD); err != nil {
+			panic(err)
+		}
+	}
+	if data, err := testFiles.ReadFile("test/pd_jsonld_jwt.json"); err != nil {
+		panic(err)
+	} else {
+		if err = json.Unmarshal(data, &pdJSONLDorJWT); err != nil {
+			panic(err)
+		}
+	}
+	if data, err := testFiles.ReadFile("test/pd_jwt.json"); err != nil {
+		panic(err)
+	} else {
+		if err = json.Unmarshal(data, &pdJWT); err != nil {
+			panic(err)
+		}
+	}
+}
+
 func TestMatch(t *testing.T) {
-	id1 := ssi.MustParseURI("1")
-	id2 := ssi.MustParseURI("2")
-	id3 := ssi.MustParseURI("3")
-	id4 := ssi.MustParseURI("4")
-	vc1 := vc.VerifiableCredential{ID: &id1}
-	vc2 := vc.VerifiableCredential{ID: &id2}
-	vc3 := vc.VerifiableCredential{ID: &id3}
-	vc4 := vc.VerifiableCredential{ID: &id4}
-	verifiableCredential := credential.ValidNutsOrganizationCredential(t)
+	// Create a JSON-LD VC
+	jsonldVC := credential.ValidNutsOrganizationCredential(t)
+
+	// Create a JWT VC
```

Comment:
JWT vc creation could be placed next to jsonld vc creation in the credential package.